### PR TITLE
Add python3 support and centos 8 support

### PR DESCRIPTION
**Pull request by funzoneq**
I fixed a print statement, so the script now works under python2 and python3. I also copied it to centos/8 and fixed the sssd-autoconfig file.

Now also included a debian 10 version.

### CODE REVIEW

File: linux/debian/10/foxpass_setup.py [hunk 1]
```diff
@@ -0,0 +1,258 @@
+#!/usr/bin/python
```
Comment (aren) [inline linux/debian/10/foxpass_setup.py]: Should we specify python3, or is that the default on debian10?

File: linux/debian/10/foxpass_setup.py [hunk 2]
```diff
@@ -0,0 +1,258 @@
...
+
+
+# check file permissions (mask should be in octal short hand ie 0644 or 0600)
+def check_perms(file, mask):
+    # first item in the set is the permissions, then we just want the last 3 digits (of the octal)
+    file_mask = oct(os.stat(file)[0])[-3:]
+    if int(file_mask, 8) == mask:
+        return True
+    else:
+        print 'updating %s to %s' % (file, oct(mask))
```
Comment (aren) [inline linux/debian/10/foxpass_setup.py]: Probably should add `()`s (or remove this line) for python3 compatibility.

File: linux/debian/10/foxpass_setup.py (hunk 1 above)
Comment (ttheune) [reply linux/debian/10/foxpass_setup.py]: We should specify python3

Comment (funzoneq) [reply linux/debian/10/foxpass_setup.py]: I made it python3 by default.

**Comment by funzoneq** [comment]
Can one of you guys merge this? Thanks!

**Pull request merged**